### ISSUE TITLE
`build` has changed the ProjectBuilder API in 1.0.0

### ISSUE THREAD

**Issue by wosc** [zestsoftware/zest.releaser#433]
```
File "/srv/jenkins/.local/share/virtualenvs/bin-RMo-sxFa/lib/python3.11/site-packages/zest/releaser/release.py", line 150, in _upload_distributions
    builder = ProjectBuilder(srcdir=".", runner=_project_builder_runner)
TypeError: ProjectBuilder.__init__() got an unexpected keyword argument 'srcdir'
```

AFACT this happend in [`build` 1.0.0](https://github.com/pypa/build/blob/main/CHANGELOG.rst#100-2023-09-01), or more precisely in [this PR](https://github.com/pypa/build/pull/537/files#diff-931e692d2041fcdba4897141e238544b6e0cc9cf429e27269fcf00fe34d796adL243-R151)

Pinning `build < 1.0.0` can be done as a workaround for now, but I guess zest.releaser will need to update its usage.

**Comment by matthewhegarty** [comment]
I had to add this to my requirements file: `zest.releaser < 9.0.0`

**Comment by reinout** [comment]
I've released `9.1.0` with the fix.
